From 8e05f006415d1811785ab5dab1ea4ae5a44c184e Mon Sep 17 00:00:00 2001
From: Karl Berry <karl@freefriends.org>
Date: Sun, 26 Jan 2020 18:06:04 -0800
Subject: [PATCH 25/25] doc: clarify build tree location

* doc/automake.texi (VPATH Builds): more words about the build
tree location. Suggestion from Jefferson Carpenter, 31 May 2018:
<https://lists.gnu.org/archive/html/automake-patches/2018-05/msg00003.html>
---
 doc/automake.texi | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/doc/automake.texi b/doc/automake.texi
index 4b26677c1..9f4acf6d8 100644
--- a/doc/automake.texi
+++ b/doc/automake.texi
@@ -860,13 +860,14 @@ Manual}, for more information about this feature.
 @cindex trees, source vs.@: build
 
 The GNU Build System distinguishes two trees: the source tree, and
-the build tree.
+the build tree.  These are two directories that may be the same, or
+different.
 
-The source tree is rooted in the directory containing
-@file{configure}.  It contains all the sources files (those that are
+The source tree is rooted in the directory containing the
+@file{configure} script.  It contains all the source files (those that are
 distributed), and may be arranged using several subdirectories.
 
-The build tree is rooted in the directory in which @file{configure}
+The build tree is rooted in the current directory at the time @file{configure}
 was run, and is populated with all object files, programs, libraries,
 and other derived files built from the sources (and hence not
 distributed).  The build tree usually has the same subdirectory layout
@@ -880,8 +881,8 @@ installation example (@pxref{Basic Installation}).
 
 A common request from users is that they want to confine all derived
 files to a single directory, to keep their source directories
-uncluttered.  Here is how we could run @file{configure} to build
-everything in a subdirectory called @file{build/}.
+uncluttered.  Here is how we could run @file{configure} to create
+everything in a build tree (that is, subdirectory) called @file{build/}.
 
 @example
 ~ % @kbd{tar zxf ~/amhello-1.0.tar.gz}
-- 
2.25.0

